src/common/slack.js

```diff
@@ -110,7 +113,7 @@ export async function submitExperimentDataRequest(accessionCode, values) {
               ]
             : []),
         ],
-        footer: `Refine.bio | ${ip} | ${navigator.userAgent}`,
+        footer: `Refine.bio | ${await getIP()} | ${navigator.userAgent}`,
```

Comment:
I would put this in a variable to make it more explicit, but it's alright like this as well.